docs/modeling/index.rst

```diff
@@ -10,337 +10,121 @@ Introduction
-    gg_fit = fitter(gg_init, x, y)
-
-    # Plot the data with the best-fit model
-    plt.figure(figsize=(8,5))
-    plt.plot(x, y, 'ko')
-    plt.plot(x, gg_fit(x), 'r-', lw=2)
-    plt.xlabel('Position')
-    plt.ylabel('Flux')
+* **Constraining parameters**
 
-This works for 1-D models, 2-D models, and combinations thereof, though there
-are some complexities involved in correctly matching up the inputs and outputs
-of all models used to build a compound model.  You can learn more details in
-the :doc:`compound-models` documentation.
+  Users can constrain parameters to be within a certain range or tie them with
+  other parameters using custom functions when fitting models to data. See
+  :ref:`here <constraining_parameters>` to view examples of how to modify and
+  constrain parameters.
 
+Fitting
+-------
+
+Fitters are classes that combine optimization algorithms with statistic
+functions to fit user data to models. They can be called as functions by
+passing in user data, model data and a model instance. See detailed
+documentation on how to fit data to models :doc:`here <fitting>`. Additional
+features regarding fitters include:
+
+* **Creating custom fitters**
+
+  Users have the ability to create their own fitters by defining an error
+  function and the type of constraints that work with their fitter (if any).
+  See documentation :ref:`here<custom-fitter>`.
+
+* **Using custom statistic functions**
+
+  Users can also implement their own statistic functions to be used with
+  their custom fitters. See documentation :ref:`here<custom-statistic>`.
+
+Reference/API
+=============
 
-Using `astropy.modeling`
-========================
+ A detailed description of the classes and functions available in the package
+ can be found :doc:`here <api>`.
+
+.. note::
+
+    `astropy.modeling` is currently a work-in-progress, and thus it is likely
+    there will still be API changes in later versions of Astropy. Backwards
+    compatibility support between versions will still be maintained as much as
+    possible, but new features and enhancements are coming in future versions.
+    If you have specific ideas for how it might be improved, feel free to let
+    us know on the `astropy-dev mailing list`_ or at
+    http://feedback.astropy.org
+
+`astropy.modeling` documentation
+================================
 
 .. toctree::
    :maxdepth: 1
 
+   quick-tutorial
    models
    parameters
    fitting
    compound-models
    new
    algorithms
-
-
-Reference/API
-=============
-
-.. automodapi:: astropy.modeling
-.. automodapi:: astropy.modeling.mappings
-.. automodapi:: astropy.modeling.functional_models
-.. automodapi:: astropy.modeling.powerlaws
-.. automodapi:: astropy.modeling.polynomial
-.. automodapi:: astropy.modeling.projections
-.. automodapi:: astropy.modeling.rotations
-.. automodapi:: astropy.modeling.fitting
-.. automodapi:: astropy.modeling.optimizers
-.. automodapi:: astropy.modeling.statistic
-
```

Comment:
Overall I like what you've done with the intro page to shorten and reorganize it.  I would still like to see one or two examples on the intro page though, so that when a user clicks on it from the main table of contents they immediately see visually what it is that this package can be used for.  To that end I really liked the examples that we did have on there.  Maybe we could shorten the example though or only show one of them (the 2D one is more eyecatching I think).  But I don't know--does anyone else have opinions on this?